### PR TITLE
Added armv5 and armv7 debian stretch images (closest to raspberry pi OS)

### PR DESCRIPTION
**Pull request by justinboswell**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/create-channel.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ jobs:
         - al2012-x64
         - al2-x64
         - ubuntu-16-x64
+        - debian-stretch-armv7hf
```
Comment (graebm) [inline .github/workflows/create-channel.yml:73]: the "hf" stands for "hard float? right?
is this docker image guaranteed to run with that exact emulation?

Comment (graebm) [reply .github/workflows/create-channel.yml:73]: The base docker images is just labeled `arm32v7`

**Comment by graebm** [comment]
The build is failing for your new image.
I made a PR once upon a time https://github.com/awslabs/aws-crt-builder/pull/61/files that would try to create new ECR repos for new images, but didn't have a way to test it so never tried to merge it
You could try merging it now and see if it fixes the issue

File: .github/workflows/create-channel.yml (hunk 1 above)
Comment (graebm) [reply .github/workflows/create-channel.yml:73]: Raspberry Pi is armv7l or armv6l

Comment (graebm) [reply .github/workflows/create-channel.yml:73]: ugh. I did `apt show --installed` on my Raspberry Pi 3B+ and noticed most packages are labeled `armhf`

I don't understand any of this stuff

Comment (justinboswell) [reply .github/workflows/create-channel.yml:73]: > the "hf" stands for "hard float? right?
> is this docker image guaranteed to run with that exact emulation?

Yes



> Raspberry Pi is armv7l or armv6l

We can add an armv6l separately

**Comment by justinboswell** [comment]
> The build is failing for your new image.
> I made a PR once upon a time https://github.com/awslabs/aws-crt-builder/pull/61/files that would try to create new ECR repos for new images, but didn't have a way to test it so never tried to merge it
> You could try merging it now and see if it fixes the issue

Yeah, the create-channel job is not resilient to a branch that doesn't exist. I'll try merging your ECR changes to main, then down and resurrect this after.

**Comment by justinboswell** [comment]
> The build is failing for your new image.
> I made a PR once upon a time https://github.com/awslabs/aws-crt-builder/pull/61/files that would try to create new ECR repos for new images, but didn't have a way to test it so never tried to merge it
> You could try merging it now and see if it fixes the issue

This is done.

**Comment by justinboswell** [comment]
> > The build is failing for your new image.
> > I made a PR once upon a time https://github.com/awslabs/aws-crt-builder/pull/61/files that would try to create new ECR repos for new images, but didn't have a way to test it so never tried to merge it
> > You could try merging it now and see if it fixes the issue
> 
> Yeah, the create-channel job is not resilient to a branch that doesn't exist. I'll try merging your ECR changes to main, then down and resurrect this after.

I also fixed this.

File: .github/workflows/create-channel.yml [hunk 2]
```diff
@@ -67,9 +67,12 @@ jobs:
         - manylinux2014-x86
         - manylinux2014-x64
         - manylinux2014-aarch64
+        #- manylinux2014-armv7l
```
Comment (graebm) [inline .github/workflows/create-channel.yml:70]: just remove this

File: .github/workflows/create-release.yml [hunk 3]
```diff
@@ -58,9 +58,12 @@ jobs:
         - manylinux2014-x86
         - manylinux2014-x64
         - manylinux2014-aarch64
+        #- manylinux2014-armv7l
```
Comment (graebm) [inline .github/workflows/create-release.yml:61]: remove

File: .github/workflows/docker-images.yml [hunk 4]
```diff
@@ -32,9 +32,12 @@ jobs:
         - manylinux2014-x86
         - manylinux2014-x64
         - manylinux2014-aarch64
+        #- manylinux2014-armv7l
```
Comment (graebm) [inline .github/workflows/docker-images.yml:35]: remove

**Review by graebm — APPROVED** [review]

**Pull request merged**